proposal.md

```diff
@@ -1,31 +1,31 @@
 # Proposal
 
-When a user says:
-My bot will respond by:
+When a user says: tell me a sports score
+My bot will respond by: looking at espn.com and giving that score back
 
-When a user says:
-My bot will respond by:
+When a user says: tell me a joke 
+My bot will respond by: tell the user a random joke
```

Comment:
This is great! You can put together a list of a few different jokes and the code can randomly choose between them.